PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -1,5 +1,7 @@
 package com.microsoft.azuretools.azureexplorer.forms;
 
+import com.microsoft.azuretools.telemetry.AppInsightsClient;
+import com.microsoft.intellij.hdinsight.messages.HDInsightBundle;
```

Comment:
Fixed

---
In reply to: [120863475](https://github.com/Microsoft/azure-tools-for-java/pull/603#discussion_r120863475) [](ancestors = 120863475)